db/seeds/UsersSeeder.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+use Phinx\Seed\AbstractSeed;
+
+class UsersSeeder extends AbstractSeed
+{
+    public function run()
+    {
+        $data = [
+            [
+                'id' => 1,
+                'name' => 'Bob Burnquist',
+                'email' => 'bob@phalconphp.com',
+                'password' => '$2a$08$Lx1577KNhPa9lzFYKssadetmbhaveRtCoVaOnoXXxUIhrqlCJYWCW',
```

Comment:
Can we add a comment on what encoded password is used? It will be helpful for our users when testing with the app.

Reply:
The problem, that I don't know =)

Reply:
Haha lol, let's see if we can generate some new passwords before going to release.

Reply:
Added in #107